homeassistant/components/heos/media_player.py

```diff
@@ -115,6 +129,11 @@ def __init__(self, player):
         """Mute the volume."""
         await self._player.set_mute(mute)
 
+    async def async_select_source(self, source):
+        """Select input source."""
+        await self.hass.data[HEOS_DOMAIN][DATA_SOURCE_MANAGER].play_source(
```

Comment:
It's easier to read if we save the manager as an instance attribute on the entity.

Reply:
Done!